bot.py

```diff
@@ -1828,8 +1828,9 @@ def build_help():
 		wa = db.get_address(user.id)
 	else:
 		wa = db.get_address(user_id)
+		user = db.get_user_by_id(user_id)
 	if wa is not None:
-		await post_dm(ctx.message.author, "Address for user: {0}", wa)
+		await post_dm(ctx.message.author, "Address for user '{0}' with Discord id {1}: {2} https://creeper.banano.cc/explorer/account/{2}", user.user_name, user.user_id, wa)
```

Comment:
This works fine if a raw user id is used, but the discord.Member object doesn't have the same members as our database Model. Until maybe an eventual schema re-write

e.g.

db.User has the fields "user_name" and "user_id"

discord.Member has the fields "name" and "id"

I think you can do something like this in a fancy pythony 1-liner

user.name if isinstance(user, discord.Member) else user.user_name, user.id if isinstance(user, discord.Member) else user.user_id